src/FeedbackBot/Views/Home/Index.cshtml

```diff
@@ -37,7 +39,7 @@
                         <span>@issue.body</span><br />
                         <span><b>Voters: </b>@issue.stringOfVoters</span>
                     </div>
-                    <div><a href="/issues/@issue.number">View Comments</a></div>
+                    <div><a href="/issues/@ViewData["AppName"]/@issue.number">View Comments</a></div>
```

Comment:
Check out the url action helpers:
href="@Url.Action("issues", new { appName = ViewData["AppName"], id = issue.number })"